.github/actions/repo-info/action.yml

```diff
@@ -25,7 +25,7 @@ inputs:
     description: 'The owners github repository ie, "trafficcontrol"'
     required: true
   branch:
-    description: 'The repository branch to query ie, "8.1.0-branch"'
+    description: 'The repository branch to query ie, "9.1.3-branch"'
```

Comment:
if `9.1.3-branch` is the only allowed value, why is this even an input?

Reply:
I think its because this action _can_ be used for other workflows and should remain dynamic for future cases

Reply:
I think this is supposed to be "e.g.", not "ie (sic)"